README.md

```diff
@@ -152,7 +152,7 @@ running by other teams.
 ## Future Work
 
 - Fine-grained authorization and auditing
-- Secure Istio components (mixer, discovery service, etc.)
+- Secure Istio components (Mixer, Istio-Manager, etc.)
```

Comment:
How about 

Istio Mixer, Istio Manager, etc.

to be consistent with the terms used previously in this doc.